prow/cluster/jobs/istio/bots/istio.bots.yaml

```diff
@@ -0,0 +1,54 @@
+presubmits:
+
+  istio/bots:
+
+    - name: bots-lint
+      path_alias: istio.io/bots
+      decorate: true
+      always_run: true
+      spec:
+        containers:
+          - image: gcr.io/istio-testing/circleci:2019-04-22
+            command:
+              - make
+              - lint
+            env:
+              - name: GO111MODULE
+                value: "on"
+        nodeSelector:
+          testing: test-pool
+
+    - name: bots-build
+      path_alias: istio.io/bots
+      decorate: true
+      always_run: true
+      spec:
+        containers:
+          - image: gcr.io/istio-testing/circleci:2019-04-22
```

Comment:
nit: Are these custom images necessary? Can you instead use the `golang` image?

Reply:
Right now, this is a lateral move from the existing CircleCi-based jobs. This image is used by other prow jobs too at this point.

We've embarked on a grand container image cleanup. We will likely be consolidating a lot of our images into a small number, used across all repos. This is starting to happen now, should be done in a few weeks I expect.